webextensions/api/tabs.json

```
@@ -2902,7 +2899,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤61"
+                    "version_added": "47"
```

:x: https://github.com/chromium/chromium/commit/8b605f21c02e8e920dcd7b26aa9716bb8ed751ad in [46.0.2484.0](https://github.com/chromium/chromium/releases/tag/46.0.2484.0) before 46.0.2490.

```suggestion
                    "version_added": "46"
```